cmd/device-util/driver.go

```diff
@@ -0,0 +1,168 @@
+//go:build windows
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"time"
+
+	cli "github.com/urfave/cli/v2"
+	"golang.org/x/sys/windows"
+)
+
+const pollTick = 5 * time.Millisecond
+
+// flags
+var (
+	_driverName string
```

Comment:
Prefer we avoid using globals for command line parameters, we can just access them via `cli.String("name")`, etc.